mlflow/transformers/__init__.py

```diff
@@ -1143,6 +1149,62 @@ def _write_card_data(card_data, path):
             )
 
 
+def _list_repository_contents(model_name):
+    """Returns the top-level file inventory of `RepoFile` objects from the huggingface hub"""
+    try:
+        import huggingface_hub as hub
+    except ImportError:
+        _logger.debug(
+            f"Unable to list repository contents for the model repo {model_name}. In order "
+            "to enable repository listing functionality, please install the huggingface_hub "
+            "package by running `pip install huggingface_hub>0.10.0"
+        )
+        return
+
+    try:
+        files = hub.list_repo_tree(model_name)
+        return list(files)
```

Comment:
```suggestion
        return next(f.path for f in files if f.path in _LICENSE_FILE_NAMES, None)
```

I'd make this function return a license file.

Reply:
good idea. Changing to the first found if exists instead.